backend/src/MessageHandler/OrderLog/CreateOrderLogMessagesHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\MessageHandler\OrderLog;
+
+use App\Message\OrderLog\OrderLogCreateMessage;
+use App\Repository\OrderEntityRepository;
+use App\Repository\StoreOwnerBranchEntityRepository;
+use App\Repository\SupplierProfileEntityRepository;
+use App\Service\OrderLog\OrderLogToMySqlService;
+use Symfony\Component\Messenger\Handler\MessageSubscriberInterface;
+
+class CreateOrderLogMessagesHandler implements MessageSubscriberInterface
+{
+    private OrderLogToMySqlService $orderLogToMySqlService;
+    private OrderEntityRepository $orderEntityRepository;
+    private StoreOwnerBranchEntityRepository $storeOwnerBranchEntityRepository;
+    private SupplierProfileEntityRepository $supplierProfileEntityRepository;
+
+    public function __construct(OrderLogToMySqlService $orderLogToMySqlService, OrderEntityRepository $orderEntityRepository,
+                                StoreOwnerBranchEntityRepository $storeOwnerBranchEntityRepository, SupplierProfileEntityRepository $supplierProfileEntityRepository)
+    {
+        $this->orderLogToMySqlService = $orderLogToMySqlService;
+        $this->orderEntityRepository = $orderEntityRepository;
+        $this->storeOwnerBranchEntityRepository = $storeOwnerBranchEntityRepository;
+        $this->supplierProfileEntityRepository = $supplierProfileEntityRepository;
+    }
+
+    public static function getHandledMessages(): iterable
+    {
+        yield OrderLogCreateMessage::class => [
+            'method' => 'handleCreateOrderLogMessage'
+        ];
+    }
+
+    public function handleCreateOrderLogMessage(OrderLogCreateMessage $orderLogCreateMessage)
+    {
+        $orderEntity = $this->orderEntityRepository->findOneBy(['id'=>$orderLogCreateMessage->getOrderId()]);
+
+        if ($orderEntity) {
+            $storeBranchEntity = $orderLogCreateMessage->getStoreOwnerBranch();
```

Comment:
if you wanna keep preparing data in this class, please move this process to a single method.